### DISCUSSION TITLE
n8n support?

### DISCUSSION THREAD

**Discussion by Morriz** [Arize-ai/phoenix#5219] [Ideas]
Hi, would you be willing to make contact with the n8n team to integrate your solution into it?

**Comment by axiomofjoy** [comment]
Hey @Morriz, we'd love to answer your questions and provide support in the #phoenix-support channel in the [Arize AI Slack community](https://join.slack.com/t/arize-ai/shared_invite/zt-2t1rdnepu-o_bGb_SQrM3QRRKxSdkiow) and in GitHub via issues and discussions.

**Reply by mkotzerke** [reply]
@axiomofjoy Would also love to have n8n integration. Cant access the slack community. Is there any plans for supporting it in the future already ?

**Reply by axiomofjoy** [reply]
Hey @mkotzerke, thanks for reaching out. I've filed an issue in our OpenInference repo [here](https://github.com/Arize-ai/openinference/issues/2048). Please feel free to upvote and add any color on what you are hoping to see.

**Comment by mikeldking** [comment]
we're willing to support this given we get buy-in from n8n. See https://community.n8n.io/t/capturing-n8n-flows-with-observability-evaluations-platforms-like-langfuse-comet-opik-etc/75764/9
